### DISCUSSION TITLE
Why are API inputs not validated like UI inputs?

### DISCUSSION THREAD

**Discussion by darkylmnx** [directus/directus#17409] [Archive]
I created a fresh new project on Directus, and I don't understand why the input sent to the API is not validated as configured in the Directus app. 

Is there a valid reason not to validate the data, although when added through the Directus app it is actually validated?

**Comment by rijkvanzanten** [comment]
Care to elaborate on what validation is missing? Validation rules configured in permissions or field settings are handled through both the app and API. Interface options only apply to the interface though.

**Reply by darkylmnx** [reply]
Well it's a validation in the field setting as shown in the image below.
![image](https://user-images.githubusercontent.com/1426357/217307762-b377190f-fc42-41be-91f0-98512f5297f1.png)
Despite that, I could do a POST request with an empty payload and it accepted with empty value.
![image](https://user-images.githubusercontent.com/1426357/217308257-67dd88d2-972d-4787-bfb2-b286201e5507.png)

**Reply by darkylmnx** [reply]
Just a reminder in case my previous message was missed.
